### PR TITLE
Sunrise edits

### PR DESCRIPTION
**Pull request by MichaelMacaulay**

### CODE REVIEW

File: website/pages/en/deploying/deploying-a-subgraph-to-hosted.mdx [hunk 1]
```diff
@@ -272,15 +272,15 @@ Graph Node exposes a graphql endpoint which you can query to check the status of
 
 This will give you the `chainHeadBlock` which you can compare with the `latestBlock` on your subgraph to check if it is running behind. `synced` informs if the subgraph has ever caught up to the chain. `health` can currently take the values of `healthy` if no errors occurred, or `failed` if there was an error which halted the progress of the subgraph. In this case, you can check the `fatalError` field for details on this error.
 
-## Hosted service subgraph archive policy
+## hosted service subgraph archive policy
```
Comment (benface) [inline website/pages/en/deploying/deploying-a-subgraph-to-hosted.mdx:275]:
```suggestion
## Hosted service subgraph archive policy
```

File: website/pages/en/sunrise.mdx [hunk 2]
```diff
@@ -1,21 +1,39 @@
...
-### What are the phases of the sunrise of decentralized data?
+### What are the phases of the Sunrise of Decentralized Data?
 
 **Sunray**: Enable support for hosted service chains, offer a seamless upgrade flow, offer a free plan on The Graph Network.  
 **Sunbeam**: The upgrade window that subgraph developers will have to upgrade their subgraphs to The Graph Network. The start and length of this will be announced soon.  
 **Sunrise**: Hosted service endpoints will expire as traffic moves to The Graph Network.
 
+### How do I upgrade my hosted service subgraph?
+
+> Note: subgraphs upgraded to The Graph Network will not be available to revert back to the hosted service.
```
Comment (benface) [inline website/pages/en/sunrise.mdx:21]: Just a wording suggestion:
```suggestion
> Note: Upgrading a subgraph to The Graph Network cannot be undone.
```

File: website/pages/en/sunrise.mdx [hunk 3]
```diff
@@ -1,21 +1,39 @@
...
+
+1. Select the subgraph(s) you want to upgrade.
+2. Select the receiving wallet (the wallet that will become the owner of the subgraph).
+3. Click the "Upgrade" button.
+
+Once your subgraph is published, the [upgrade Indexer](#what-is-an-upgrade-indexer) will begin serving queries on it, so you can begin making queries immediately!
+
+### What happens if I don't upgrade my subgraph?
+
+Hosted service endpoints will expire at the end of the upgrade window.
```
Comment (benface) [inline website/pages/en/sunrise.mdx:35]: Would be nice to have more details about when that is?

Comment (MichaelMacaulay) [reply website/pages/en/sunrise.mdx:35]: CC: @azf20 are we clear to include a date for the upgrade window in tomorrow's docs release, or should we hold off on that? That date is not mentioned in the [blog](https://docs.google.com/document/d/1TIX0dHtoWDfIJsu9EArtYU-Rw5s6S0Jb6OBmj9hCk1s/edit) or social media copy.

File: website/pages/en/billing.mdx [hunk 4]
```diff
@@ -8,7 +8,18 @@ There are two plans to use when querying subgraphs on The Graph Network.
...
 - **Free Plan**: The Free Plan includes 100,000 free monthly queries with full access to the Subgraph Studio testing environment. This plan is designed for hobbyists, hackathoners, and those with side projects to try out The Graph before scaling up.
 
-- **Growth Plan**: The Growth Plan includes everything in the Free Plan with all queries after 100,000 monthly queries requiring payments with GRT or credit card (coming soon). The Growth Plan is flexible enough to cover teams that have established dapps across a variety of use cases.
+- **Growth Plan**: The Growth Plan includes everything in the Free Plan with all queries after 100,000 monthly queries requiring payments with GRT or credit card. The Growth Plan is flexible enough to cover teams that have established dapps across a variety of use cases.
+
+<VideoEmbed youtube="Nn771WagX2I" />
+
+## Query Payments with credit card
+
+- To set up billing with Stripe, users will access Subgraph Studio (https://thegraph.com/studio/)
```
Comment (azf20) [inline website/pages/en/billing.mdx:17]: I don't think "billing with Stripe" is the most user-friendly way to position this, this should be framed as fiat payments by credit or debit card.

File: website/pages/en/billing.mdx [hunk 5]
```diff
@@ -8,7 +8,18 @@ There are two plans to use when querying subgraphs on The Graph Network.
...
+    1. Go to the [Subgraph Studio Billing page](https://thegraph.com/studio/billing/).
+    2. Click on the "Connect Wallet" button on the top right corner of the page. You'll be redirected to the wallet selection page. Select your wallet and click on "Connect".
+    3. Choose “Upgrade plan” if you are upgrading from the Free Plan or choose “Manage Plan” if you have already added GRT to your billing balance in the past. Next, you can estimate the number of queries to get a pricing estimate, but this is not a required step. 
+    4. To choose a credit card payment, choose “Credit card” as the payment method and fill out your credit card information. Those who have used Stripe before can use the Link feature to autofill their details. 
+- Stripe invoices will be processed at the end of each month and require an active credit card on file for all queries beyond the free plan quota.
```
Comment (azf20) [inline website/pages/en/billing.mdx:22]: I think this can just say "Invoices" (not Stripe invoices)

File: website/pages/en/sunrise.mdx (hunk 3 above)
Comment (azf20) [reply website/pages/en/sunrise.mdx:35]: This will be announced at a later date

File: website/pages/en/billing.mdx [hunk 6]
```diff
@@ -26,8 +37,6 @@ The Graph’s billing system accepts GRT from either Arbitrum or Ethereum, but u
 
 ### Adding GRT using a wallet
 
-<VideoEmbed youtube="4Bw2sh0FxCg" />
```
Comment (azf20) [inline website/pages/en/billing.mdx:29]: Is this video no longer applicable?

File: website/pages/en/cookbook/subgraph-debug-forking.mdx [hunk 7]
```diff
@@ -44,12 +44,12 @@ export function handleUpdatedGravatar(event: UpdatedGravatar): void {
 }
 ```
 
-Oops, how unfortunate, when I deploy my perfect looking subgraph to the [Hosted Service](https://thegraph.com/hosted-service/) it fails with the _"Gravatar not found!"_ error.
+Oops, how unfortunate, when I deploy my perfect looking subgraph to the [hosted service](https://thegraph.com/hosted-service/) it fails with the _"Gravatar not found!"_ error.
```
Comment (azf20) [inline website/pages/en/cookbook/subgraph-debug-forking.mdx:47]: There are quite a few edits which are capitalisation changes?
I think we need to update examples etc. to use the Studio and Network

**Review by azf20 — CHANGES_REQUESTED** [review]
Thanks!

File: website/pages/en/cookbook/subgraph-debug-forking.mdx (hunk 7 above)
Comment (MichaelMacaulay) [reply website/pages/en/cookbook/subgraph-debug-forking.mdx:47]: My thought process here is that we can change the hosted service capitalization now, and then when Sunbeam starts, we can replace hosted service with studio. Afaik, we are entering something of a liminal space between Sunray's completion and the Sunbeam phase, so I'd imagine we should keep hosted service in docs for now?

File: website/pages/en/billing.mdx (hunk 6 above)
Comment (MichaelMacaulay) [reply website/pages/en/billing.mdx:29]: CC: @brandonleekramer

**Review by azf20 — APPROVED** [review]
Thanks!

**Pull request merged**